data/json/mutations/mutations.json

```diff
@@ -1955,35 +1955,87 @@
     "valid": false,
     "vitamin_rates": [ [ "instability", -60 ] ]
   },
+  {
+    "type": "mutation",
+    "id": "JAUNDICE",
+    "name": { "str": "Jaundice" },
+    "points": 0,
+    "visibility": 1,
+    "ugliness": 1,
+    "description": "It's subtle, but your skin has definitely taken on a sickly yellow-green hue.",
+    "types": [ "SKIN", "BODY_ARMOR", "ARMOR", "BODY_ARMOR_MOD" ],
+    "changes_to": [ "PLANTSKIN" ],
+    "category": [ "PLANT", "ELFA" ]
+  },
   {
     "type": "mutation",
     "id": "SKIN_ROUGH",
-    "name": { "str": "Rough Skin" },
+    "name": { "str": "Goosebumps" },
     "points": 0,
     "visibility": 2,
     "ugliness": 1,
-    "description": "Your skin is slightly rough.  This has no gameplay effect.",
-    "types": [ "SKIN" ],
-    "changes_to": [ "SPARSE_SCALES", "FEATHERS", "LIGHTFUR", "CHITIN", "PLANTSKIN", "PATCHSKIN1" ],
-    "category": [
-      "LIZARD",
-      "RAPTOR",
-      "BIRD",
-      "SPIDER",
-      "MOUSE",
-      "PLANT",
-      "CHIMERA",
-      "FISH",
-      "BEAST",
-      "CATTLE",
-      "RAT",
-      "FELINE",
-      "LUPINE",
-      "RABBIT",
-      "INSECT",
-      "ELFA",
-      "URSINE",
-      "CRUSTACEAN"
+    "description": "You have goosebumps that never seem to go away.  Are they getting bigger?",
+    "types": [ "SKIN", "BODY_ARMOR", "ARMOR", "BODY_ARMOR_MOD" ],
+    "changes_to": [ "FEATHERS" ],
+    "category": [ "BIRD" ]
+  },
+  {
+    "type": "mutation",
+    "id": "SKIN_RASHY",
+    "name": { "str": "Rashy Skin" },
+    "points": -1,
+    "visibility": 2,
+    "ugliness": 2,
+    "description": "You've developed a persistent skin rash.  Sometimes it's fine, other times it itches like hell.",
+    "types": [ "SKIN", "BODY_ARMOR", "ARMOR", "BODY_ARMOR_MOD" ],
+    "changes_to": [ "PATCHSKIN1" ],
+    "category": [ "CHIMERA" ]
+  },
+  {
+    "type": "mutation",
+    "id": "HIRSUTE",
+    "name": { "str": "Hirsute" },
+    "points": 0,
+    "description": "Your body hair is growing in thicker than it used to.  It's almost like you're going through a second puberty or something.",
+    "types": [ "SKIN", "BODY_ARMOR", "ARMOR", "BODY_ARMOR_MOD" ],
```

Comment:
What's the benefit of grabbing all these mutation types?  Nothing currently uses this combination.

Reply:
That's the entire point of the PR. Without those types attached, you suffer the problem described in the associated issue, where you can, for instance, lose CHITIN1 but keep CHITIN2, which is unintended and leads to all kinds of problems.

Most of ths mutations using those flags are designed to assume that there's a specific mutation type beneath them. If you'rs still not following, please follow the steps in the attached issue to see what the original problem was and then try that again with these changes.

Reply:
I forgot that no one has moved the major body transformations behind the Thresholds.  That was discussed previously and would have made this less necessary. At least in some cases.